pywsitest/utils.py

```diff
@@ -1,23 +1,51 @@
 import re
+from typing import List, Tuple
 
 
 PATH_REGEX = re.compile(r"^\$\{(.*)\}$")
 
 
-def get_resolved_value(response: dict, path: str) -> object:
+def get_resolved_values(response: [list, dict], path: str) -> List[object]:
     """
-    Retrieves a value from a dictionary at a given path
+    Retrieves a list of values from a dictionary at a given path
 
     Parameters:
-        response (dict): The response to check against for a value
-        path (str): The path in the response to check for a value
+        response (list, dict): The response to check against for values
+        path (str): The path in the response to check for values
 
     Returns:
-        (object): The object at a given path, or None if the path can't be found
+        (list[object]): The list of objects at a given path, empty if the path can't be found
     """
-    resolved = response
-    for part in path.split("/"):
-        if part not in resolved:
-            return None
-        resolved = resolved[part]
+    resolved = [response]
```

Comment:
can response not be a list itself?

Reply:
I cant seem to find a test where response is a list.

Reply:
also, looking at the code, I cant find a case where response is a list

Reply:
Changed the method signature, response now has to be a dictionary.
Added #36 as a feature request for this.